--- build/l4t/source/public/kernel/kernel-4.9/scripts/Kbuild.include	2022-08-01 00:00:00.000000000 +0000
+++ build/l4t/source/public/kernel/kernel-4.9/scripts/Kbuild.include	2022-08-01 12:00:00.000000000 +0000
@@ -461,8 +461,8 @@
 # It's a common trick to declare makefile variable that contains space
 # we'll need it to convert the path string to list (string delimited by spaces)
 # and vice versa
-the-space :=
-the-space +=
+E =
+the-space = $E $E
 # TEGRA_ROOT_PATH is the relative path to the directory one level upper than $srctree
 _TEGRA_ROOT_PATH = $(subst ^$(realpath $(srctree)/..)/,,^$(realpath $(kbuild-dir)))
 # _TEGRA_REL_PATH is path like "../../../" that points to directory one level
